cmd/core/main.go

```diff
@@ -172,6 +174,15 @@ func main() {
 		setupLog.Error(err, "unable to setup the vela core controller")
 		os.Exit(1)
 	}
+	if driver := os.Getenv("STORAGEDRIVER"); len(driver) == 0 {
```

Comment:
please use a const variable for `STORAGEDRIVER `

Reply:
Done!